dev-ruby/pusher-client/pusher-client-0.6.2.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
+RUBY_FAKEGEM_RECIPE_DOC="rdoc"
+RUBY_FAKEGEM_RECIPE_TEST="rspec"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Client for consuming WebSockets from http://pusher.com"
+HOMEPAGE="https://github.com/pusher-community/pusher-websocket-ruby"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+ruby_add_bdepend "
+	dev-ruby/bundler
```

Comment:
In general we try to avoid build-time dependencies like bundler since often complicates longer-term maintenance. We usually patch this out in all_ruby_prepare(), e.g.:

```
sed -i -e '/bundler/ s:^:#:' Rakefile || die
```